aspnetcore/razor-pages/sdk.md

```diff
@@ -55,6 +55,16 @@ To use the Razor SDK to build class libraries containing Razor views or Razor Pa
 
 ::: moniker-end
 
+## Razor language version
+
+The Razor parser and code generator version is controlled by the `<RazorLangVersion>` MSBuild property. If the app requires a different Razor language version than the version of the shared framework used by the app, set the version in the app's project file:
```

Comment:
Hmm, today if a user has a mismatched `RazorLangVersion` 99.99% of the time it's because of mismatched SDK + project or incompatible VS bits.

I don't know if I'd suggest people modify the `RazorLangVersion` directly. @pranavkm I believe you added this error message. Is there a better way we can phrase this or is having a user manually set the `RazorLangVersion` appropriate?

Reply:
I could reword it to say that **_in rare cases_** an app requires this. Something like ...

> The Razor parser and code generator version is controlled by the `<RazorLangVersion>` MSBuild property. In rare cases, an app requires a different Razor language version than the version of the shared framework used by the app. The version is set in the app's project file:

Reply:
... and another thing we could do is move this new section all the way to the end of the topic to minimize its significance to most scenarios.

Reply:
> I could reword it to say that **_in rare cases_** an app requires this.

That would be beneficial but i'm still not convinced it's ever necessary pending response from @pranavkm. If it is necessary then yes, i'd rephrase to say **rare** 



> and another thing we could do is move this new section all the way to the end of the topic to minimize its significance to most scenarios.

Assuming it's necessary this would be perfect.

Reply:
The problem was the link ... Check out that issue https://github.com/aspnet/AspNetCore.Docs/issues/12142 ... an error link lands here with no coverage.